src/gui/torrentcontentmodelfile.cpp

```diff
@@ -69,7 +72,10 @@ void TorrentContentModelFile::setPriority(BitTorrent::DownloadPriority newPriori
 void TorrentContentModelFile::setProgress(qreal progress)
 {
     m_progress = progress;
-    m_remaining = static_cast<qulonglong>(m_size * (1.0 - m_progress));
+    if (m_priority == BitTorrent::DownloadPriority::Ignored)
+        m_remaining = 0;
+    else
+        m_remaining = static_cast<qulonglong>(m_size * (1.0 - m_progress));
```

Comment:
I would use `?:` operator.

Reply:
Or what about making the changes to `TorrentContentModelItem::remaining()`? All cases can be covered there, isn't?